### ISSUE TITLE
Channel Context-Menu doesn't work on Firefox Android

### ISSUE THREAD

**Issue by infinite-persistence** [lbryio/lbry-desktop#6582]
<img src="https://user-images.githubusercontent.com/64950861/126323287-12f18f7f-14dc-43d5-8412-caec2b5e789c.png" width="400">

Nothing happens.  I was logged in -- not sure if that mattered.

**Comment by btzr-io** [comment]
Works fine for me.  ( Latest version of firefox android: 90.1.1 )

**Comment by infinite-persistence** [comment]
Seems to work in general but fails on the "The Kurt Angle Show" channel.

**Comment by infinite-persistence** [comment]
Closed by #6470
